src/garage/tf/baselines/gaussian_mlp_baseline.py

```diff
@@ -5,10 +5,10 @@
 
 from garage import make_optimizer
 from garage.np.baselines import Baseline
+from garage.tf.baselines.gaussian_mlp_regressor_model import (
```

Comment:
similarly -- can you inline this class and rename it `_GaussianMLPBaselineModel`?